app/Models/Foundation/Summit/Registration/SummitTicketType.php

```diff
@@ -69,6 +69,14 @@ class SummitTicketType extends SilverstripeBaseModel
         self::Audience_Without_Invitation,
     ];
 
+    const Subtype_Regular = 'regular';
```

Comment:
constants should be start with UpperCase ( Regular)

Reply:
done